CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 5.1.6
+ - Docs: Fix broken link by removing extra space. 
```

Comment:
Would you mind changing this to use the format described in [the logstash plugin contribution guidelines](https://github.com/elastic/logstash/blob/master/CONTRIBUTING.md#what-is-the-format-of-the-changelog-entries) for writing changelogs?